### PR TITLE
feat: add Skeleton (#353)

### LINKED ISSUE #353: ImageAlt can be relpaced with a Skeleton
### Description

Image Alternative text shows up before the Avatar is rendered, The Alternate text can be replaced with the `Skeleton` of Prime react 
So the user is able to see a skeleton rather than that Alternate text and after the avatar is rendered the skeleton disappears for it.

### Screenshots

Screenshot 2021-09-29 at 12 58 16 PM

### Additional information

I have tried solving this issue but facing a problem while solving it and will be needing some help with it

### PR DESCRIPTION
**Pull request by Guneetsinghtuli**
resolve #353 

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/383"></a>

### CODE REVIEW

**Comment by kkhitesh** [comment]
It would be nice, if you added screenshots depicting the change

**Comment by schmelto** [comment]
if updated the r description to link the issue

**Comment by Guneetsinghtuli** [comment]
I have Attached the Screen recording Please have a look

https://user-images.githubusercontent.com/48324492/135596312-d475ce33-a762-44de-984e-30b93141cc95.mov

File: src/Components/Home.js [hunk 1]
```diff
@@ -1,12 +1,17 @@
 import './Home.css'
-
```
Comment (krishguptadev) [inline src/Components/Home.js:2]: Is there a reason for removing this line?

**Review by krishguptadev — CHANGES_REQUESTED** [review]

File: src/Components/Home.js (hunk 1 above)
Comment (Guneetsinghtuli) [reply src/Components/Home.js:2]: @krishguptadev No, There is no reason as such, But if you want I can add it again.

**Review by AbhijithGanesh — CHANGES_REQUESTED** [review]
Is there a reason as to why a certain number of profiles are excluded in your video? If that is happening to the main project, it'd be unfair. Can you work around this and patch the issue?

**Review by AbhijithGanesh — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]

File: src/Components/Home.js (hunk 1 above)
Comment (krishguptadev) [reply src/Components/Home.js:2]: I think that it helps separate the imports and it looks pleasing to the eye to

**Comment by Guneetsinghtuli** [comment]
@krishguptadev I have made the changes that you suggested

File: src/Components/Home.js [hunk 2]
```diff
@@ -19,20 +25,51 @@ function Home() {
       .finally(() => setShowProgress(false))
   }, [])
 
+  //  Get Avatar
+  useEffect(() => {
+    list.forEach((user) => {
+      const splitId = user.avatar.split('/')
+      const userID = splitId[3].split('.')
+      const githubID = userID[0]
+      const url = 'https://avatars.githubusercontent.com/' + githubID
```
Comment (eddiejaoude) [inline src/Components/Home.js:34]: not everyone will have images from GitHub

File: src/Components/Home.js [hunk 3]
```diff
@@ -22,6 +24,18 @@ function Home() {
   return (
     <main>
       {showProgress && <ProgressBar mode="indeterminate" />}
+      {skeleton
+        ? <div>
+        <div style={{ display: 'none' }}>
+      {setTimeout(() => {
+        setskeleton(false)
+      }, 3000)}
```
Comment (eddiejaoude) [inline src/Components/Home.js:32]: Can this be moved outside the html?

Comment (eddiejaoude) [reply src/Components/Home.js:32]: 3000ms (3s) is too long, I think 300ms is enough

File: src/Components/Home.js [hunk 4]
```diff
@@ -22,6 +24,18 @@ function Home() {
...
+      </div>
+      <div className="p-d-flex p-flex-wrap">
+          {list.map((user, index) => { return <Skeleton shape="circle" size="65px" className="p-mr-2 p-m-2" key={index} /> })}
+      </div>
+      </div>
+        : <>
```
Comment (eddiejaoude) [inline src/Components/Home.js:38]: the large ternary is difficult to read, it might be easier to separate into functions

**Review by eddiejaoude — APPROVED** [review]
Following our discussion on Discord, I know you spent a lot of time on this, so I won't write more comments, I will merge as it is and make changes

File: src/Components/Home.js (hunk 3 above)
Comment (krishguptadev) [reply src/Components/Home.js]: I think 350ms would be better

Comment (AbhijithGanesh) [reply src/Components/Home.js]: 300ms would be enought!

**Pull request merged**